packages/vats/scripts/init-network.js

```diff
@@ -0,0 +1,18 @@
+import { makeHelpers } from '@agoric/deploy-script-support';
+
+export const defaultProposalBuilder = async ({ publishRef, install }) =>
```

Comment:
I sure wish we had static types here... or some documentation on the shape of this thing. not critical.

I suppose it's an argument to `writeCoreProposal` which is a product of `makeHelpers`, which is imported. So there _is_ a follow-your-nose path, provided you're used to our define-before-use style.